<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLogin</class>
 <widget class="QMainWindow" name="TelaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logar</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../Imagens/icone.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="Logar">
   <widget class="QPushButton" name="PlanodeFundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>871</width>
      <height>519</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/grr.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>850</width>
      <height>600</height>
     </size>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="User">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>58</y>
      <width>169</width>
      <height>125</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/user.png</normalon>
      <disabledoff>../Imagens/user.png</disabledoff>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>0</number>
    </property>
    <property name="autoRepeatInterval">
     <number>0</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Login">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>336</y>
      <width>165</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_Login.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>163</width>
      <height>163</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cadastro">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>338</y>
      <width>165</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../Imagens/btn_cadastro.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="IconeSair">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <disabledon>../Imagens/Icone sair.png</disabledon>
      <selectedon>../Imagens/Icone sair.png</selectedon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Box_ExcluirCadastros">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>394</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(0, 22, 57);
text-decoration: underline;
font: 75 8pt &quot;Microsoft YaHei&quot;;
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtTop</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Excluir cadastros</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>312</y>
      <width>239</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 12pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>*Usuário e/ou senha incorretos*</string>
    </property>
   </widget>
   <widget class="QLabel" name="SenhaMestre">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>432</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
font: 75 10pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="text">
     <string>Senha mestre:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Ir">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>428</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Century Gothic&quot;;
color: rgb(0, 22, 57);
background-color: rgb(220, 20, 60);</string>
    </property>
    <property name="text">
     <string>Ir</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>470</y>
      <width>173</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 12pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>*Senha mestre incorreta*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Login_Usuario">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>212</y>
      <width>375</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Usuário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Login_Senha">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>268</y>
      <width>375</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Login_SenhaMestre">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>428</y>
      <width>185</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>PlanodeFundo</zorder>
   <zorder>User</zorder>
   <zorder>btn_Login</zorder>
   <zorder>btn_Cadastro</zorder>
   <zorder>IconeSair</zorder>
   <zorder>label</zorder>
   <zorder>SenhaMestre</zorder>
   <zorder>btn_Ir</zorder>
   <zorder>Box_ExcluirCadastros</zorder>
   <zorder>label_2</zorder>
   <zorder>Login_Usuario</zorder>
   <zorder>Login_Senha</zorder>
   <zorder>Login_SenhaMestre</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
